pages/admin/settings/sso-connection/edit/[id].tsx

```diff
@@ -0,0 +1,46 @@
+import type { GetServerSidePropsContext, NextPage } from 'next';
+import useSWR from 'swr';
+import { useRouter } from 'next/router';
+
+import { fetcher } from '@lib/ui/utils';
+import EditConnection from '@components/connection/EditConnection';
+import { serverSideTranslations } from 'next-i18next/serverSideTranslations';
+import Loading from '@components/Loading';
+import { errorToast } from '@components/Toaster';
+import type { ApiError, ApiSuccess } from 'types';
+import type { OIDCSSORecord, SAMLSSORecord } from '@boxyhq/saml-jackson';
+
+const EditSSOConnection: NextPage = () => {
```

Comment:
@niwsa `Allowed redirect URLs` and `Default redirect URL` are non-editable fields on the connection create page but are editable on the connection edit page. Should it be consistent?

Reply:
Good spot. I have pushed a fix which fixes the redirection on edit button click.